i18n/de.toml

```diff
@@ -45,41 +45,41 @@ other = "Challenge"
 other = "Granny Alert"
 
 [challenges_saw_description]
-other = """How can we improve the responsiveness to emergencies of elderly people within their homes without compromising their privacy?
+other = """Wie k&ouml;nnen wir die Reaktionsf&auml;higkeit auf Notf&auml;lle &auml;lterer Menschen zu Hause verbessern, ohne ihre Privatsph&auml;re zu gef&auml;hrden?
 
-Provided by Stiftung Alterswohnungen"""
+Zur Verf&uuml;gung gestellt von Stiftung Alterswohnungen"""
 
 [challenges_taz1_title]
 other = "Move and Chill"
 
 [challenges_taz1_description]
-other = """Help us to improve knowledge by measuring the utilization of movable chairs in the city.
+other = """Hilf uns, unser Wissen zu verbessern, indem du die Nutzung von beweglichen St&uuml;hlen in der Stadt misst.
 
-Provided by Civil Engineering Office"""
+Zur Verf&uuml;gung gestellt vom B&uuml;ro f&uuml;r Bauingenieurwesen"""
 
 [challenges_ttn_title]
 other = "Breathe!"
 
 [challenges_ttn_description]
-other = """Empower citizens to map and understand air pollution in the city.
+other = """Erm&ouml;gliche den Bewohnern, die Luftverschmutzung in der Stadt zu erfassen und zu verstehen.
```

Comment:
**Bewohner*innen**

Please check if there are any other pronouns that need to be made gender-neutral.

Reply:
thx, fixed now, and I didn't find others...